lib/trino-filesystem-azure/src/test/java/io/trino/filesystem/azure/AbstractTestAzureFileSystem.java

```diff
@@ -129,11 +109,10 @@ void afterEach()
     private void cleanupFiles()
     {
         if (accountKind == AccountKind.HIERARCHICAL) {
-            DataLakeFileSystemClient fileSystemClient = new DataLakeFileSystemClientBuilder()
-                    .endpoint("https://%s.dfs.core.windows.net".formatted(account))
-                    .fileSystemName(containerName)
-                    .credential(credential)
-                    .buildClient();
+            DataLakeServiceClientBuilder serviceClientBuilder = new DataLakeServiceClientBuilder()
+                    .endpoint("https://%s.dfs.core.windows.net".formatted(account));
+            azureAuth.setAuth(account, serviceClientBuilder);
+            DataLakeFileSystemClient fileSystemClient = serviceClientBuilder.buildClient().getFileSystemClient(containerName);
```

Comment:
The reason is because I changed the builder type from `DataLakeFileSystemClientBuilder` -> `DataLakeServiceClientBuilder` in order to be able to call `azureAuth.setAuth(account, serviceClientBuilder)` since the `AzureAuth` interface only has methods for `DataLakeServiceClientBuilder` and `BlobContainerClientBuilder`. The `DataLakeServiceClientBuilder` doesn't have a method to set the `.fileSystemName(...)`